### PR TITLE
Refine

### PR DESCRIPTION
**Pull request by ghost**
added different kind of metrics.

followup work would be to measure the client start/stop time, zio start/stop time, though it is already at the point to use a profiler

### CODE REVIEW

File: src/main/scala/App.scala [hunk 1]
```diff
@@ -17,8 +17,8 @@ import zio.http.netty.client.NettyClientDriver
 import java.net.URI
 import java.net.http.{HttpClient, HttpRequest, HttpResponse}
 
-
-val urlHttp = "http://google.com"
+//for now, ensure not http requests are used
```
Comment (jamesward) [inline src/main/scala/App.scala:20]: The http tests were there to check for https overhead.  We should leave that in.

File: src/main/scala/App.scala [hunk 2]
```diff
@@ -103,7 +103,7 @@ object App extends ZIOAppDefault:
         jdkReq(urlHttps, client)
       }
     }
-
+/*
```
Comment (jamesward) [inline src/main/scala/App.scala:106]: Let's leave these tests in.

File: src/main/scala/App.scala [hunk 3]
```diff
@@ -137,16 +137,19 @@ object App extends ZIOAppDefault:
     }
 
     for
+      /*
```
Comment (jamesward) [inline src/main/scala/App.scala:140]: Leave these in.

File: src/main/scala/App.scala [hunk 4]
```diff
@@ -159,5 +162,47 @@ object App extends ZIOAppDefault:
...
       _ <- printLine(s"10 https zio fixed pool = ${tenPool._1.dividedBy(10).toMillis}ms / req")
+
+      _ <- printLine("\n--------------------------------------------------")
+      _ <- printLine("TOTAL: contains client&ZIO startup/shutdown, similar to cli launch")
+      _ <- printLine("Diff : difference between 1st and 2nd request")
+
+      totalTime <- req2(urlHttps, "ZIO Client.default").provide(Client.default).timed
+      _ <- printLine(s"TOTAL = ${totalTime._1.toMillis}ms")
+
+      totalTime <- req2(urlHttps, "ZIO Client pooled").provide(Client.default).timed
```
Comment (jamesward) [inline src/main/scala/App.scala:173]: What is the default for pools in `Client.default`?  We should probably put a bit more on that in the description.

Comment (ghost) [reply src/main/scala/App.scala:173]: I do not analyze details at this point, things change with newer versions, then the fine-tuning comes.

**Comment by ghost** [comment]
You do not need to merge this, but note that I cannot maintain 2 versions (the one merged here and the one on my fork). There is usually always a reason for a change, even if it is to just reduce complexity for now to get this working with snapshots). The main relevant change here is the report of diff/total.

File: src/main/scala/App.scala (hunk 1 above)
Comment (ghost) [reply src/main/scala/App.scala:20]: can be added later, currently just too much.

**Comment by ghost** [comment]
As an alternative processing, you could just update your code to use and updated version (snapshot), see https://github.com/jamesward/zio-http-client-perf/issues/3, and keep the rest as is.

**Comment by ghost** [comment]
closing as outdated.

**Pull request closed without merging**